### PR TITLE
update ghci container to restore gh/ci

### PR DESCRIPTION
**Pull request by mahf708**
updates ghci-aci container and its corresponding machine entry.

[BFB]

### CODE REVIEW

File: .github/workflows/e3sm-gh-ci-cime-tests.yml [hunk 1]
```diff
@@ -61,7 +57,7 @@ jobs:
         with:
           name: ${{ matrix.test }}
           path: |
-            ~/projects/e3sm/scratch/${{ matrix.test }}*/TestStatus.log
-            ~/projects/e3sm/scratch/${{ matrix.test }}*/bld/*.bldlog.*
-            ~/projects/e3sm/scratch/${{ matrix.test }}*/run/*.log.*
-            ~/projects/e3sm/scratch/${{ matrix.test }}*/run/*.cprnc.out
+            /projects/e3sm/scratch/${{ matrix.test }}*/TestStatus.log
```
Comment (bartgol) [inline .github/workflows/e3sm-gh-ci-cime-tests.yml:60]: I'm assuming this uploads the test results somewhere so we can inspect them in case of failure? Would it be bad to simply cat the files so we just see them in the action output? Or is retrieving them easy from the web interface?

**Review by bartgol — APPROVED** [review]

File: .github/workflows/e3sm-gh-ci-cime-tests.yml (hunk 1 above)
Comment (bartgol) [reply .github/workflows/e3sm-gh-ci-cime-tests.yml:60]: Ah, I suppose cat-ing so many files may make the output hard to parse, given that we run a matrix of tests... So maybe yeah, uploading them is cleaner.

**Review by rljacob — APPROVED** [review]

File: .github/workflows/e3sm-gh-ci-cime-tests.yml (hunk 1 above)
Comment (bartgol) [reply .github/workflows/e3sm-gh-ci-cime-tests.yml:60]: Long shot, but we could cat them only if there's a fail maybe? We could run create test step by step. In pseudo code:
```yaml
   - create_test ${{ matrix.test }} --no-setup
   - cd $output_root/${{ matrix.test }}*
   - case.setup || cat TestStatus.log
   - case.build || cat bld/*.bldlog.*
   - case.submit || cat run/e3sm.log.*
```
It is not correct/enough, since a) ERP/PEM tests create case2run and case2bld folders, and b) the fail at submit may come from cprnc, not actual run. 
I guess it would be nice if CIME supported catting the correct file in case of a test fail. E.g., something like `create_test <TEST> --output-on-failure` or something, would output the bldlog/log/cprnc.out of the step that failed...

Comment (rljacob) [reply .github/workflows/e3sm-gh-ci-cime-tests.yml:60]: I'm not sure what you mean.  cat them to the screen?   If you run create_test interactively, it will tell you where to look when there's a fail.  Catting a file to a screen would just scroll hundreds of lines of text on your screen.

Comment (bartgol) [reply .github/workflows/e3sm-gh-ci-cime-tests.yml:60]: Yes, I meant to cat them to screen, so that they are readily visible on the action output on gh. But I don't know if one can scroll back to very old aciton runs or if those also expire after 90 days. If they expire, then at this point, there's not much added value in cat-ing them. We can just be satisfied with the logs uploaded as artifacts ...

**Comment by rljacob** [comment]
@mahf708 is this ready?
